openpype/modules/deadline/plugins/publish/submit_max_deadline.py

```diff
@@ -160,18 +159,16 @@ def process_submission(self):
         }
 
         self.log.debug("Submitting 3dsMax render..")
-        payload = self._use_published_name(payload_data)
+        project_settings = instance.context["project_settings"]
+        payload = self._use_published_name(project_settings, payload_data)
         job_info, plugin_info = payload
         self.submit(self.assemble_payload(job_info, plugin_info))
 
-    def _use_published_name(self, data):
+    def _use_published_name(self, project_settings, data):
```

Comment:
Move `project_settings` to be second argument

Reply:
There was also wrong name of variable. Fixed